MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -3,19 +3,19 @@ PODS:
     - MapboxMobileEvents (= 0.10.2)
   - MapboxAccounts (2.3.0)
   - MapboxCommon (4.0.0)
-  - MapboxCoreNavigation (1.0.0-beta.2):
```

Comment:
Turns out we forgot to update `CocoaPodsTest` in master, so this is not true cherry pick, I had to update `Podfile.lock`, because build job was [failing](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/1678/workflows/17a2b57f-b4bf-4888-9118-5eb900a021e2/jobs/20012).